rca/static_src/sass/components/_form-item.scss

```diff
@@ -362,6 +373,14 @@
         @include font-size(xxs);
     }
 
+    input,
+    textarea,
+    select {
+        &:-webkit-autofill {
```

Comment:
doesn't look like there's a `:moz` equivalent for these, guessing you don't need them then?

Reply:
as far as I can see mozilla no long offers autocomplete, so we don't need to worry about it